modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,288 @@
+
+Mule automatically logs multiple messages and specific elements in your app flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[Logger component] anywhere in a flow and set it up to output any message you want.
+
+You can view an app log as follows:
+
+* If you're running an app from the Anypoint Platform, the output from the app log is visible in Anypoint Studio's console window.
+
+* If you're using Mule from the command line to run an app, the app log is visible in your OS console.
+
+You can also view the app log in this default location, unless the log file path was customized in the app's log file (`log4j2.xml`):
+
+`MULE_HOME/logs/<app-name>.log`
+
+== Runtime Log
+
+The runtime log (`mule_ee.log`) contains information about app and lifecycle events. For example, the log records an entry when a Mule service or app starts, deploys, stops, or undeploys. The log is located in the `log4j2.xml` file in the `conf` directory. You can customize this file when running the server in standalone mode.
+
+To view the runtime log in Anypoint Studio:
+
+* Click *Anypoint Studio* -> *About Anypoint Studio* -> *Installation Details* -> *Configuration* tab -> *View Error Log*:
++
+image::logging-in-mule-view-error-log.png[View Error Log]
+
+== Configure logging
+
+You can create a configuration file to define:
+
+ * What kinds of messages to log
+ * How to log the messages (asynchronously or synchronously)
+ * Where to log the messages, such as to the console or disk, or to an endpoint or database
+
+Mule uses `slf4j`, which is a logging facade that discovers and uses a logging strategy from the classpath, such as Apache Log4j 2 or the JDK Logger. By default, Mule includes Log4j 2, which is configured with a file called `log4j2.xml`.
+
+== Synchronous Versus Asynchronous Logging
+By default, Mule logs messages asynchronously. When logging synchronously, the execution of the thread that is processing your message is interrupted to wait for the log message to be fully handled before it can continue:
+
+image::logger-synch.png[align="center"]
+
+When logging asynchronously, the logging operation occurs in a separate thread, so the message can process before the logging completes:
+
+image::logger-asynch-alternativo.png[align="center"]
+
+In most situations, MuleSoft recommends that you use *asynchronous logging* because it substantially improves the throughput and lowers the latency of message processing.
+
+When asynchronous logging is used, some actions might not be logged if there is a system crash. This situation occurs because log writing is performed on a separate thread that runs independently of other actions. See xref:exception_handling[Exception Handling with Asynchronous Logging] for information on mitigating this issue.
```

Comment:
_When asynchronous logging is used_, some actions might not be logged if there is a system crash. >
Using asynchronous logging cause that some actions may not be logged if there is a system crash

Reply:
Keeping as is - rewording is also awkward. Also, if we change it here, we probably have to go back to v4.1, which is extra work for something minor. As long as the meaning is clear... This is a sentence that Kim could probably help with in the future.